.changeset/report-cli-env-write-errors.md

```
@@ -0,0 +1,5 @@
+---
+'mastra': patch
+---
+
+Fixed CLI environment updates so callers are notified when values cannot be persisted.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)